offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/SelectItemsView.groovy

```diff
@@ -232,36 +232,43 @@ class SelectItemsView extends VerticalLayout implements Resettable{
         HorizontalLayout quantitySeq = new HorizontalLayout(amountSequencing,applySequencing)
         quantitySeq.setComponentAlignment(applySequencing, Alignment.BOTTOM_LEFT)
         VerticalLayout seqLayout = new VerticalLayout(sequencingGrid,quantitySeq)
+        sequencingGrid.setHeightByRows(6)
```

Comment:
what is this change about? how is this related to the design?

Reply:
the product grids are very high and for notebook screens, this makes the UI already unnecessary large and extended